### ISSUE TITLE
Treva Reid

### ISSUE THREAD

**Issue by sfdoran** [caciviclab/odca-jekyll#336]
Treva Reid has had her campaign committee for District 7 open since 2014, but she has never actually run. Usually the only candidate committees that cover several election cycles are those for officeholders, so we restrict the data to the election cycle.

In Treva's case this misrepresents her campaign contributions and spending by over $10,000. How should we handle Treva's campaign?

![image](https://user-images.githubusercontent.com/19939684/74288259-cf264b00-4ce0-11ea-887c-98bff9825c73.png)

**Comment by ckingbailey** [comment]
What field is it in the spreadsheet that controls the year range for which we capture a candidate's finance data? I see Treva Reid has `Start Date` "1/1/2014". That doesn't make the backend capture her contributions back to 2014?

**Comment by sfdoran** [comment]
@ckingbailey you're right it's the start date and end date on the candidate tab of the spreadsheet. It's probably resolved now.

**Comment by mikeubell** [comment]
We only download data to 2015.  Here is the results including 2014:
 "supporting_money": {                                                            
~     "contributions_received": 59699.5,                                             
~     "total_contributions": 59699.5,                                                
~     "total_expenditures": 10275.419999999998,                                      
~     "total_loans_received": 0.0,                                                   
      "total_supporting_independent": 0.0,                                           
      "support_list": [
